### ISSUE TITLE
DORIS: MEMORY LEAKS!!!!!!

### ISSUE THREAD

**Issue by Krystine** [SealHAT/GUI_Application#28]
@hpan5 There are two MAJOR memory leaks!!!!!!!!!

In both send_serialSetup and receive_serialSetup, memory for microSerial is allocated every time either function is called BUT IS NEVER DEALLOCATED. This could be why the application runs slow, as you mentioned earlier. 

Memory for this variable should be declared ONLY ONCE throughout the entire application's run-time, and needs to be deallocated somewhere when it is no longer needed!
